src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -116,14 +117,22 @@ const getHeadlineComponent = (props: ComponentToRenderProps) => (
   <ArticleHeadline {...props} />
 );
 
-const ArticlePage = ({ pageData }: { pageData: Article }) => {
-  const { isApp } = useContext(RequestContext);
+const ArticlePage = ({
+  pageData,
+  continueReadingEnabled = false, // change to true when developing to see on the page (comment can be removed when the experiment is ready to run)
```

Comment:
Would it be good to have a test on the article page for the rendering of the `read more`/`continue reading` button? Maybe an integration style render test rather than individual unit tests but that's up to you

Reply:
changes to this file are not necessary anymore as we went with a different method, so I will revert this file back to how it was before

Reply:
If the optimizely code is to be included in the ArticlePage code (I imagine it would be?) then we could add a test mocking that optimizely call to get the variation, but we can do that in the PR that adds the optimizely code. I am imagining line 122 will be removed when we add the optimizely code in this file.

If I add tests for it appearing on the expected services here, we would just need to remove that test in the next PR and replace with the one mocking optimizely.

However, unit tests could still be written for it not showing on lite site, amp, app and on desktop screen width.

Reply:
I have added some tests in article page for conditions which affect the button being rendered or not